crates/agent/src/publications/specs.rs

```diff
@@ -486,6 +488,12 @@ pub async fn add_built_specs_to_live_specs(
         {
             agent_sql::publications::add_built_specs(row.live_spec_id, &collection.spec, txn)
                 .await?;
+            agent_sql::publications::add_inferred_schema_md5(
```

Comment:
yeah, agreed. I left it this way for expediency, and because I'd like to think about addressing that a little more holistically, if there's a practical way to do so. We do _lots_ of separate insert/update statements, and it seems to be hurting readability, in addition to the performance concerns. Though I also don't want to add to that unnecessarily...